doc/source/rllib.rst

```diff
@@ -171,15 +171,18 @@ Custom Environments
 To train against a custom environment, i.e. one not in the gym catalog, you
 can register a function that creates the env to refer to it by name. The contents of the
 ``env_config`` agent config field will be passed to that function to allow the
-environment to be configured. For example:
+environment to be configured. The return type should be an `OpenAI gym.Env <https://github.com/openai/gym/blob/master/gym/core.py>`__. For example:
```

Comment:
When I view the rendered HTML, this link somehow gets abbreviated to https://github.com/openai/gym.

Reply:
Weird, it looks ok to me though.

Reply:
This is fine for both the github rendering and on my local machine; there is another link for `OpenAI Gym MDP`, so maybe you clicked on that instead?